R/archive.R

```diff
@@ -665,21 +666,22 @@ epi_archive =
  
               # Note that we've already recycled comp value to make size stable,
               # so tibble() will just recycle time value appropriately
-              return(tibble::tibble(time_value = time_value, 
+              return(tibble::tibble(version = version, 
                                     !!new_col := comp_value))
             }
             
             # If f is not missing, then just go ahead, slide by group
             if (!missing(f)) {
+              
               if (rlang::is_formula(f)) f = rlang::as_function(f)
               
-              x = purrr::map_dfr(ref_time_values, function(t) {
+              x = purrr::map_dfr(ref_versions, function(t) {
```

Comment:
Please rename `t` -> `ref_time_value` to improve consistency with the current naming approach.